src/Resources/defaultConfig/webpack.config.js

```diff
@@ -35,10 +35,12 @@ module.exports = function makeWebpackConfig(options) {
         path: options.parameters.path ? options.parameters.path : __dirname + '/../../web/compiled/',
 
         // Output path from the view of the page
-        // Uses webpack-dev-server in development
         publicPath: options.parameters.public_path
             ? options.parameters.public_path
-            : (BUILD ? '/compiled/' : 'http://localhost:8080/compiled/'),
+            : (BUILD
+              ? '/compiled/' // PROD path compile only
+              : (process.env.WEBPACK_MODE === 'watch' ? 'http://localhost:8080/compiled/' // DEV path webpack-dev-server is run
```

Comment:
I think we could totally switch just to checking `WEBPACK_MODE`. Currently we have duplicated configuration (`/compiled/` is repeated in several places, this could introduce bugs if someone modifies first occurrence but misses the second one)